packages-web/maps/webpack.config.prod.js

```diff
@@ -1,7 +1,28 @@
 const merge = require("webpack-merge");
 const baseConfig = require("../utils-react-widgets/configs/webpack.config.prod");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 const customConfig = {
+    output: {
```

Comment:
This is already handled in the webpack config of utils-react-widgets, so I think it's not necessary here.

Reply:
Check the case, this is not handled in the utils-react

Reply:
Ah yes, because of the maps-web name